src/tokens/TokenDetails.test.tsx

```diff
@@ -261,6 +263,36 @@ describe('TokenDetails', () => {
     expect(queryByText('celoNews.headerTitle')).toBeTruthy()
   })
 
+  it('renders earn card for Aave Arbitrum Usdc when user has balance', () => {
+    jest.mocked(getFeatureGate).mockReturnValue(true)
```

Comment:
can we use the gate in question here? e.g.,
```suggestion
    jest.mocked(getFeatureGate).mockImplementation(gate => gate === <required_gate>)
```
Otherwise we'll have stray mocks like the one in the below test (for blockchain api price history) left over and not cleaned up

Reply:
Fixed in 92487c6fd453b95c7dcf7d53556238e6c36d57c5!